components/release/release-team.yaml

```diff
@@ -11,3 +11,17 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: component-maintainer
+---
+kind: RoleBinding
```

Comment:
ClusterRole is cluster wide as the name state but using it in a RoleBinding grant the access to the resources only in the namespace the RoleBinding belongs to, `release-service` in this case.

If you only want to be able to view/edit `releaseserviceconfigs` in the `release-service`  namesapce, I suggest to use a Role instead of a ClusterRole. If you want to be able to view/edit  `releaseserviceconfigs` in any namespace, then use  a ClusterRoleBinding instead or a RoleBinding.

Reply:
thanks for the clarification and advice. Fixed.